crates/ibc/src/applications/transfer/context.rs

```diff
@@ -93,16 +93,16 @@ pub trait BankKeeper {
 
 /// Captures all the dependencies which the ICS20 module requires to be able to dispatch and
 /// process IBC messages.
-pub trait Ics20Context:
-    Ics20Keeper<AccountId = <Self as Ics20Context>::AccountId>
-    + Ics20Reader<AccountId = <Self as Ics20Context>::AccountId>
+pub trait TransferContext:
```

Comment:
Thinking about this some more, I would prefer the names `TokenTransfer{Context,Keeper,Reader}`. @hu55a1n1 WDYT?

Reply:
done